arch/xtensa/src/esp32/Kconfig

```diff
@@ -81,25 +81,23 @@ config ESP32_SDMMC
 	---help---
 		No yet implemented
 
-config ESP32_SPI0
```

Comment:
What's the reason to remove SPI0 and SPI1?  SPI0 is somewhat special, but the difference with SPI1 is that SPI2 and 3 can be configured as slaves.

Reply:
Yes, that was the idea, it only support SPI2 and SPI3 because these ports can be master or slave. The code should be modified in the future to support SPI0 and SPI1

Reply:
Shouldn't then only the slave source code be dependent on `CONFIG_ESP32_SPI2 || CONFIG_ESP32_ESP3`?  The master code is similar, right?

Reply:
Good point! I agree! You are right, thank you very much Abde!